src/index.html

```diff
@@ -15,8 +15,110 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,
+      300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,
+      900&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Moyo header</h1>
+
+  <body class="ttq">
+    <header>
+      <a
+        href="#moyo"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo company logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#apple"
+              class="nav_link is-active"
```

Comment:
Consider using a more descriptive class name than `nav_link`. Class names should represent the meaning of the content. For example, `navigation-link` would be more descriptive.